### DISCUSSION TITLE
Add license indicator to all entries

### DISCUSSION THREAD

**Discussion by maromiki** [kausalflow/tools#94] [Comments]
Each entry in tools.kausalflow have a list which summarizes content and properties of the entry. For instance, Curvenote has:

- Web
- General and Interdisciplinary
- Publishing and Sharing
- ...

We could improve the list by adding a license indicator (Free, Freemium, Paid, Proprietary, Open Source). This would be useful to filter out apps which require a paid license to work properly.

**Comment by emptymalei** [comment]
Hi @maromiki 
I love this idea. There was a field called "plans" to indicate the pricing of the tool. I thought it was tedious to type in such information and I never used it. The field looks like this:

```
plans:
  - name:
    description:
```

I think what you proposed is great. 


Added to todo:
https://github.com/orgs/kausalflow/projects/6/views/1?pane=issue&itemId=40438178
